include/dfrn.php

```diff
@@ -827,7 +824,7 @@ private static function get_attachment($doc, $root, $item) {
 	 * @return object XML entry object
 	 * @todo Add type-hints
 	 */
-	private static function entry($doc, $type, $item, $owner, $comment = false, $cid = 0) {
+	private static function entry($doc, $type, $item, $owner, $comment = false, $cid = 0, $single = false) {
```

Comment:
The `$single` parameter does need a description in the doxygen header

Reply:
Yeah, I will do this later today.

Reply:
Done